crates/viewer/re_view_spatial/src/view_3d_properties.rs

```diff
@@ -53,5 +53,35 @@ impl TypedComponentFallbackProvider<Plane3D> for SpatialView3D {
             .map_or(DEFAULT_PLANE, |up| Plane3D::new(up.as_vec3(), 0.0))
     }
 }
+use re_types::{blueprint::components::Eye3DKind, components::LinearSpeed};
 
-re_viewer_context::impl_component_fallback_provider!(SpatialView3D => [BackgroundKind, Color, StrokeWidth, Plane3D]);
+// Logic should be similar to `impl TypedComponentFallbackProvider<LinearSpeed> for ViewEye`
+impl TypedComponentFallbackProvider<LinearSpeed> for SpatialView3D {
+    fn fallback_for(&self, ctx: &re_viewer_context::QueryContext<'_>) -> LinearSpeed {
+        let maybe_state = re_viewer_context::ViewStateExt::downcast_ref::<crate::SpatialViewState>(
+            ctx.view_ctx.view_state,
+        );
+        let speed = match maybe_state {
+            Ok(spatial_view_state) => {
+                let bounding_boxes = &spatial_view_state.bounding_boxes;
+                match spatial_view_state.state_3d.view_eye {
+                    Some(eye) => eye.fallback_speed_for_mode(bounding_boxes) as f64,
+                    None => 1.0_f64,
+                }
+            }
+            Err(view_system_execution_error) => {
+                re_log::error!("Error while downcasting {}", view_system_execution_error);
+                1.0_f64
+            }
+        };
+        LinearSpeed(re_types::datatypes::Float64(speed))
+    }
+}
+
+impl TypedComponentFallbackProvider<Eye3DKind> for SpatialView3D {
+    fn fallback_for(&self, _ctx: &re_viewer_context::QueryContext<'_>) -> Eye3DKind {
+        Eye3DKind::default()
+    }
+}
+
+re_viewer_context::impl_component_fallback_provider!(SpatialView3D => [BackgroundKind, Color, StrokeWidth, Plane3D, LinearSpeed, Eye3DKind]);
```

Comment:
as per note on ViewEye, let's not duplicate this code and rather pass in the fallback provider to the ViewEye. Avoiding this kind of oracle code duplication is the very reason those fallback providers are there in the fist place - they provide a mechanism to share fallback code with specifc parts of the code (the camera here) and generic parts (the ui).

Reply:
I'd either have one forward to the other or just pass in the fallback provider to the camera.. although I guess that might cause a borrow problem 🤔 
I can have a look if you're done with everything else (?) :)

Reply:
sure, thing - have a look if you like.
There's just the `clone` comment above; and just a general question about `f32` and `f64` conversion back and forth (that a brought up a while a go in https://github.com/rerun-io/rerun/pull/10445#issuecomment-3026695604 )

Reply:
made the forwarding more straight forward, but yeah you were right there's only so much that can be done, I was wrong to call it duplicated code
[`20c4e27` (#10445)](https://github.com/rerun-io/rerun/pull/10445/commits/20c4e27d3d90ecba2038b841c5cb66a12d4906c2)